veba-bom.json

```diff
@@ -60,6 +60,94 @@
             }
         ]
     },
+    "knative": {
+        "gitRepoTag": "v0.20.0",
+        "containers": [{
+                "name": "gcr.io/knative-releases/github.com/projectcontour/contour/cmd/contour@sha256",
+                "version": "3e88236949d49b0933318ef077741e9bf6f40ae84e8b79d7c229e1aaff53220f"
+            },
+            {
+                "name": "envoyproxy/envoy",
+                "version": "v1.16.0"
+            },
+            {
+                "name": "gcr.io/knative-releases/knative.dev/net-contour/cmd/controller@sha256",
+                "version": "5654ef5be4f506909bba6cbb59e7049d6c4bfc866066f2546bcabd2ff547307f"
+            },
+            {
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/activator@sha256",
+                "version": "cbeb1cdcdde8b3b44bc5eb1c5a07512a36f4bd7fbee5e9677e8789ee2459b6ed"
+            },
+            {
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/autoscaler@sha256",
+                "version": "73d4d0503544ea930caa4fba07b701147339131de31a6aae8c52d6bb568f3c81"
+            },
+            {
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/controller@sha256",
+                "version": "3054c326df9de07a2a1ce34a2515339201706995a16f8efc8228295a6b94fbdd"
+            },
+            {
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/webhook@sha256",
+                "version": "07894526a4b49c61c5333e4f48190e830ce5124f0c18e1042cc15fcb594bfc37"
+            },
+            {
+                "name": "gcr.io/knative-releases/controller-17550e04ebf9af897aa4b799ad074e13@sha256",
+                "version": "641c03911641bc86fc1b70db14e55caaa6a42c83fcc4f7cc9da50622afd6566e"
+            },
+            {
+                "name": "gcr.io/knative-releases/mtping-26ac4e6a49a610ffaac30cc0a8ab693a@sha256",
+                "version": "14c4da0990f2d6a27d62fe5c1d336d2663b034fa98835c672126afd719a1b03c"
+            },
+            {
+                "name": "gcr.io/knative-releases/webhook-f31b5fdf3b3cdeb5a25aa0a5d79e4348@sha256",
+                "version": "eb480afef446b9f0b016cf82d88553c9a0adb4dde39505ca3d2e5830e79399e4"
+            }
+        ]
+    },
+    "knative-function": {
+        "gitRepoTag": "latest",
+            "containers": [{
+                "name": "embano1/kn-echo",
+                "version": "latest"
```

Comment:
`latest` is not causing any issues for your offline capabilities, right?

Reply:
Correct.